### PR TITLE
Ibp 5221 implement get locations

### PR DESCRIPTION
**Pull request by lik-83**

### CODE REVIEW

**Review by clarysabel — APPROVED** [review]

File: src/main/java/org/ibp/api/java/inventory/manager/LotTemplateExportService.java [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+import org.generationcp.middleware.api.location.LocationDTO;
 import org.ibp.api.domain.ontology.VariableDetails;
 
 import java.io.File;
 import java.util.List;
 
 public interface LotTemplateExportService {
 
-	File export(final List<LocationDto> locations, final List<VariableDetails> scales);
+	File export(final List<LocationDTO> locations, final List<VariableDetails> scales);
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/java/inventory/manager/LotTemplateExportService.java:11]: @lik-83 please take the chance to remove the finals

File: src/main/java/org/ibp/api/java/location/LocationService.java [hunk 2]
```diff
@@ -15,9 +14,10 @@
 
 	List<LocationTypeDTO> getLocationTypes();
 
-	long countLocations(String crop, LocationSearchRequest locationSearchRequest);
+	long countLocations(String crop, LocationSearchRequest locationSearchRequest, final String programUUID);
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/java/location/LocationService.java:17]: remove final @lik-83

File: src/main/java/org/ibp/api/java/location/LocationService.java [hunk 3]
```diff
@@ -15,9 +14,10 @@
...
 
-	List<LocationDto> getLocations(String crop, LocationSearchRequest locationSearchRequest, Pageable pageable);
+	List<LocationDTO> searchLocations(String crop, LocationSearchRequest locationSearchRequest, Pageable pageable,
```
Comment (clarysabel) [inline src/main/java/org/ibp/api/java/location/LocationService.java:19]: remove finals @lik-83

File: src/main/java/org/ibp/api/java/inventory/manager/LotTemplateExportService.java (hunk 1 above)
Comment (lik-83) [reply src/main/java/org/ibp/api/java/inventory/manager/LotTemplateExportService.java]: done

File: src/main/java/org/ibp/api/java/location/LocationService.java (hunk 2 above)
Comment (lik-83) [reply src/main/java/org/ibp/api/java/location/LocationService.java]: done

File: src/main/java/org/ibp/api/java/location/LocationService.java (hunk 3 above)
Comment (lik-83) [reply src/main/java/org/ibp/api/java/location/LocationService.java:19]: done

**Pull request merged**